lib/99 bottles.rb

```diff
@@ -0,0 +1,17 @@
+  def ninety_nine_bottles_of_beer
```

Comment:
Assignment Branch Condition size for ninety_nine_bottles_of_beer is too high. [27.04/15]